bin/node/browser-testing/Cargo.toml

```diff
@@ -12,7 +12,7 @@ libp2p = { version = "0.18.0", default-features = false }
 jsonrpc-core = "14.0.5"
 serde = "1.0.106"
 serde_json = "1.0.48"
-wasm-bindgen = { version = "0.2.60", features = ["serde-serialize"] }
+wasm-bindgen = { version = "0.2.62", features = ["serde-serialize"] }
```

Comment:
To fix the node browser test on CI, you should revert this change and re-run `cargo update`.